src/routes.ts

```diff
@@ -1,27 +1,69 @@
 import * as express from 'express';
 
-import { dispatchOrmUserActions } from 'models/User/controllers';
+import {
+    createTheme,
+    findTheme,
+    updateUserThemeById,
+} from 'models/UserTheme/controllers';
+import { createUser, getAllUsers } from 'models/User/controllers';
+import { createMsg, getMsgsByTopicId } from 'models/Message/controllers';
+import { createTopic, getTopics } from 'models/Topic/controllers';
 
 const router = express.Router();
 
-router.get('/orm-actions', (_, res) => {
-    // это пример использования модельки. потом можно впихивать это в какие угодно файлы
-    // удалю после апрува пр, когда скажете, что всё ок и всем всё понятно :>
-    // и да, я помню, что мы договаривались в качестве примера ничего не писать
-    // но тут, мне показалось, что надо
+router.post('/user-theme', (req, res) => {
+    const { userid } = req.body;
 
-    // это пример использования модельки. потом можно впихивать это в какие угодно файлы
-    // удалю после апрува пр, когда скажете, что всё ок и всем всё понятно :>
-    // и да, я помню, что мы договаривались в качестве примера ничего не писать
-    // но тут, мне показалось, что надо
+    findTheme(userid.toString())
+        .then((el) => res.json(el))
+        .catch(() => res.sendStatus(400));
+});
+
+router.post('/add-user', async (req, res) => {
+    const { first_name, second_name, login, email, phone, identifier } =
+        req.body;
+
+    await createUser(first_name, second_name, login, email, phone, identifier);
+    await createTheme('light', identifier);
+
+    await res.sendStatus(200);
+});
+
+router.post('/change-theme', async (req, res) => {
+    const { theme, id } = req.body;
+    const userTheme = await findTheme(id.toString());
+    updateUserThemeById(userTheme!.id, { theme });
+    res.sendStatus(200);
```

Comment:
❗
И так же по всему файлу/файлам
```suggestion
    const userTheme = await findTheme(id.toString());
    
    updateUserThemeById(userTheme!.id, { theme });
    
    res.sendStatus(200);
```